SPH_SIMULATIONS.md

```diff
@@ -0,0 +1,82 @@
+## Running an SPH Simulation
+
+This section assumes users know how to configure a SplishSplash and/or DualSPHysics
+simulation, in which case our **Inductiva API client** provides a simple interface to run
+those simulations at scale. Users are, in any case, invited to consult the relevant documentation
+for each of those simulation engines (links
+[here](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH) and
+[here](https://github.com/DualSPHysics/DualSPHysics), respectively). 
+
+Let's demonstrate how to use two SPH simulators with **Inductiva API**.
+
+### SplishSplash
+
+SplishSplash is simple to run and the simulator is simply configured via a few configuration files.
+The required one is a `.json` file that contains the parameters of the simulation.
+The others are extra input files that determine the geometry and properties of the
+objects and fluids in the simulation.
+
+```python
+import inductiva
+
+input_dir = inductiva.utils.files.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "splishsplash-input-example.zip", unzip=True)
+
+# Set simulation input directory
+splishsplash = inductiva.simulators.SplishSplash()
+
+task = splishsplash.run(input_dir=input_dir,
+                        sim_config_filename="config.json")
+
+# Wait for the simulation to finish
+task.wait()
+task.download_outputs()
+```
```

Comment:
````suggestion

1. Inspect the content of the input folder for this simulation:

```bash
$ ls -l1 sph_demos/simple_splishsplash
config.json
unit_box.obj
```

2. Inspect the content of the python script:

```python
$ cat simple_splishsplash.py
import inductiva

# Instantiate a SPlisHSPlasH proxy
splishsplash = inductiva.simulators.SplishSplash()

# submit the task to the remote execution pipeline, indicating 
# the path to the local directory containing the required input files
# and the name of the configuration file
task = splishsplash.run(input_dir='sph_demos/splishsplash',
                        sim_config_filename="config.json")

# wait for the simulation to finish
task.wait()

# and download all generated output files
task.download_outputs()
```

3. Running the python script:

```bash
$ python simple_splishsplash.py
2023-12-20 14:51:06 INFO Task ID: 1703083865802376474
2023-12-20 14:51:06 INFO Packing input archive to upload to inductiva.ai.
2023-12-20 14:51:06 INFO Uploading packed input archive with size 1.07 KB.
2023-12-20 14:51:06 INFO Input archive uploaded.
2023-12-20 14:51:06 INFO Task submitted to the default resource pool.
2023-12-20 14:51:21 INFO Task completed successfully.
2023-12-20 14:51:30 INFO Downloading simulation outputs to inductiva_output/1703083865802376474/output.zip.
100%|██████████████████████| 261k/261k [00:00<00:00, 5.15MB/s]
2023-12-20 14:51:31 INFO Uncompressing the outputs to inductiva_output/1703083865802376474.
```

4. Inspecting the output of the simulation in `inductiva_output/1703083865802376474`:

```bash
$ ls -l1 inductiva_output/1703083865802376474
log
stderr.txt
stdout.txt
vtk
```

````